samples/hssi/hssi_200g_400g_cmd.h

```diff
@@ -0,0 +1,310 @@
+// Copyright(c) 2020-2023, Intel Corporation
```

Comment:
```suggestion
// Copyright(c) 2023, Intel Corporation
```
Just 2023, because it's a new file.  Range dates indicate that the file has been around for multiple years.

Reply:
Thanks. Updated after running clang-format and fixing the date.

Reply:
https://intelpedia.intel.com/Kernel_New_Code_Tips_and_Tricks#Documentation,
Please refer,  Copyright(c) 2023, Intel Corp.

Reply:
Sorry, but this is incorrect.  We've had precedent set on this copyright format since the predecessor to OPAE and we've adhered to it strictly since that time.  What you're proposing would make these 2 files different than every other file in the source base.  Not something we want to do.